docs/ref/files/file.txt

```diff
@@ -56,8 +56,10 @@ The ``File`` class
         was originally opened with; ``None`` means to reopen with the original
         mode.
 
-        Returns ``self``, so that it can be used similar to Python's
-        built-in :func:`python:open()` with the ``with`` statement.
+        .. versionchanged:: 2.0
+
+            Returns ``self``, so that it can be used similar to Python's
```

Comment:
I think the "returns self" phrase is an implementation detail that doesn't need to be mentioned. Would it be more straightforward to say something like "Can be used as a context manager, e.g. `with file.open() as f: ...`.

The `versionchanged` annotation should be "self-contained" as described in https://docs.djangoproject.com/en/dev/internals/contributing/writing-documentation/#documenting-new-features.